View/media_control_panel.py

```diff
@@ -11,3 +12,18 @@ def __init__(self):
         to the panel using some layout.
         """
         super().__init__()
+        
+        # Create play pause btn
+        self.play_pause_button = QPushButton()
+        self.play_pause_button.setIcon(self.style().standardIcon(QStyle.SP_MediaPlay))
+        
+        #Create horizontal box layout
```

Comment:
Add a space after the "#" symbol, and add a punctuation mark at the end of the comment. Ex.
# Creates a horizontal box layout.